src/app/pages/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,31 @@
+import WelcomePageIcon from '../../components/Welcome/WelcomePageIcon.svg';
+import { FormEvent, useState } from 'react';
+import styles from './WelcomePage.module.css';
+import Button from '../../components/Button/Button';
+
+function WelcomePage(): JSX.Element {
+  const [name, setName] = useState('');
+
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+  }
+
+  return (
+    <div className={styles.container}>
+      <img className={styles.icon} src={WelcomePageIcon} alt="" />
+      <form className={styles.form} onSubmit={handleSubmit}>
+        <input
+          className={styles.textInput}
+          type="text"
+          placeholder="my name"
+          value={name}
+          maxLength={12}
+          onChange={(event) => setName(event.target.value)}
+        ></input>
+        <Button className={''} type={'submit'} value={'enter'} />
```

Comment:
Did you want to give a className here?

Reply:
I need the class name for other buttons in my design but I can't ignore the className here, therefor an empty string.